.rustfmt.toml

```diff
@@ -10,10 +10,10 @@ use_small_heuristics = "Off"
 # Unstable options we would like to use in future
 # -----------------------------------
 
-#imports_granularity = "Module"
+imports_granularity = "Module"
 #blank_lines_lower_bound = 1
-#group_imports = "One"
+group_imports = "One"
```

Comment:
Can we use `StdExternalCrate` instead of `One`? This will group imports first by std library imports, then the next group is external crates, and the last one is internal crate imports. It makes it really easy, at a glance to see what external dependencies you're using.

```suggestion
group_imports = "StdExternalCrate"
```